lib/util/fs/index.ts

```diff
@@ -160,6 +174,18 @@ export async function localPathIsFile(pathName: string): Promise<boolean> {
   }
 }
 
+export async function localPathIsSymbolicLink(
+  pathName: string
+): Promise<boolean> {
+  const path = ensureLocalPath(pathName);
+  try {
+    const s = await fs.lstat(path);
+    return s.isSymbolicLink();
```

Comment:
```suggestion
    return s?.isSymbolicLink() ?? false;
```
lstat can return `null` or `undefined, so we should be a little defense here instead of throwing a null reference exception (perf)

Reply:
how will this happen? I ask because I couldn't find a way to setup test for the branch coverage.

Reply:
use `jest.spyOn(fs, 'lstat').mockResolvedValueOnce()` in a test